src/Libraries/RevitNodes/Application/Document.cs

```diff
@@ -84,19 +86,25 @@ public DynamoUnits.Location Location
         /// <summary>
         /// Saves all of the input families at a given location.
         /// </summary>
-        /// <param name="family">The families to save</param>
-        /// <param name="directoryPath">Directory to save the families in</param>
+        /// <param name="family">The Revit family to save</param>
+        /// <param name="directoryPath">Directory to save the family to. If directory does not exist, it will be created.</param>
         /// <returns>File path of saved families</returns>
-        public string SaveFamilyLibrary(Family family, string directoryPath)
+        public string SaveFamilyLibraryToFolder(Family family, string directoryPath)
```

Comment:
don't think we need `Library` in the name any more since in Revit that refers to saving all families in one operation. Let's remove here.